### ISSUE TITLE
Unsigned numpy uint64 unsupported

### ISSUE THREAD

**Issue by suleimanmashini** [plotly/plotly.py#1155]
Hi, When I used a pandas array of uint64 on plotly, the numbers appear negative.
I know that they cannot be as i force the type to unsigned when i append:
tempDataFrame[YaxisIn].astype(np.uint64)
In this call:
                traces.append(go.Scattergl(
                    x = tempDataFrame[XaxisIn],
                    y = tempDataFrame[YaxisIn].astype(np.uint64),
                    mode = 'markers',
                    marker = dict (line = dict(width = 1)),
                    name = database + "." + TablesIn + ".notUseful"
                    )
                )

**Comment by jonmmease** [comment]
Hi @suleimanmashini,

Please provide a full example that demonstrates the issue, with data and imports, formatted in a "fenced code block" https://www.google.com/url?sa=t&source=web&rct=j&url=https://help.github.com/articles/creating-and-highlighting-code-blocks/&ved=2ahUKEwjMq--AgJ_dAhWQl-AKHSQrCG4QFjAAegQICBAD&usg=AOvVaw297etkhd2-RtKM-CXyUZ9h.

Also please include the version of plotly.py you are using.

Thanks!

**Comment by suleimanmashini** [comment]
Hi @jonmmease , 
ofcourse Heres a code block that recreates the problem. I'm currently using plotly 3.0.2. I appreciate the help!
```
import plotly.graph_objs as go
import plotly
import pandas as pd
import numpy as np

print(plotly.__version__)

d = { 'xaxis': [1,2,3,4], 'yaxis': [4173500704,4147505216,4147504160,4143500800]}

df = pd.DataFrame(data=d, dtype=np.uint64)

traces = []

print(df.dtypes)

traces.append(go.Scattergl(
    x = df['xaxis'],
    y = df['yaxis'],
    mode = 'markers',
    marker = dict (line = dict(width = 1)),
    name = "sample cata"
    )
)

plotly.offline.plot({
    "data": traces
}, auto_open=True)

```

**Comment by suleimanmashini** [comment]
This is what I get when running the sample commented above:
![screen shot 2018-09-04 at 09 52 49](https://user-images.githubusercontent.com/33000035/45021034-6d13e180-b028-11e8-9a17-a10ba5713530.png)

**Comment by jonmmease** [comment]
Hi @suleimanmashini ,

Could you try again with plotly.py 3.2.0 (released yesterday)?  When I run your example with this version I don't see this issue:

![newplot](https://user-images.githubusercontent.com/15064365/45158442-0e7a6f00-b1b2-11e8-96c1-e8c63ffa86d5.png)

If that doens't fix it for you, can you post what version of numpy you have installed?

I did notice that trying to plot a figure with a `uint64` array breaks the `FigureWidget` serialization, so I'm going to mark this as a bug either way.

**Comment by suleimanmashini** [comment]
Hey,

updating it fixed the problem for me. Thanks!

**Comment by jonmmease** [comment]
The `FigureWidget` issue noted above has been fixed in #1163
